sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -1751,4 +1736,34 @@ public Response<Void> publishComponentTelemetryWithResponse(String digitalTwinId
     }
 
     //endregion TelemetryAPIs
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<DigitalTwinsBulkJob> createBulkImportJob(String id, DigitalTwinsBulkJob importJob) {
```

Comment:
Please double check the naming here with Rachel and Abhinav. I believe Saad wanted to stay away from the word "Bulk" but call it an "Import Job" instead. This is how the naming was also done in .NET.

Reply:
Thanks- removed the Bulk from the function.